docs/release.md

```diff
@@ -47,6 +47,27 @@ git push upstream ${RELEASE_TAG}
 
 This will trigger a [release GitHub action](https://github.com/rancher/cluster-api-provider-rke2/blob/main/.github/workflows/release.yml) that creates a release with RKE2 provider components.
 
+## Post release operations
```

Comment:
It is pre-release action. `metadata.yaml` needs to be included into release payload. If it happens after, another release is needed

Reply:
Thanks for reporting this. It is fixed now.